Source/DafnyLanguageServer/Language/DafnyLangParser.cs

```diff
@@ -72,9 +73,23 @@ public sealed class DafnyLangParser : IDafnyParser, IDisposable {
         if (parseErrors != 0) {
           logger.LogDebug("encountered {ErrorCount} errors while parsing {DocumentUri}", parseErrors, document.Uri);
         }
+
         if (!TryParseIncludesOfModule(program.DefaultModule, program.BuiltIns, errorReporter, cancellationToken)) {
           logger.LogDebug("encountered error while parsing the includes of {DocumentUri}", document.Uri);
         }
+
+        return program;
+      } catch (Exception e) {
+        logger.LogDebug("encountered an exception while parsing {DocumentUri}", document.Uri);
+        logger.LogDebug(e.ToString());
+        var t = new Token {
+          filename = document.GetFilePath(),
+          line = 1,
+          col = 1,
+          pos = 0,
+          val = "."
```

Comment:
Does this token string have any special meaning? Would an empty string be valid?

Reply:
It could be any character. It has to be at least one character long, else the error would not appear on the VSCode report.

Reply:
Ah, okay. If the purpose is to make it show up in VSCode, a more meaningful name would probably make sense (which I think is the same as @camrein's suggestion).

Reply:
The thing is, I don't want this to be displayed more than one character, since it's an internal exception. I changed it to a space though.